diff --git a/dist/index.d.mts b/dist/index.d.mts
index 65bd7878e96059f976844b5eefe8a256ff48d40d..99c7228e7d10f640fb8af63521e5faecec9ae4a4 100644
--- a/dist/index.d.mts
+++ b/dist/index.d.mts
@@ -1264,5 +1264,5 @@ declare function apiKey(_configurations?: (ApiKeyConfigurationOptions & ApiKeyOp
   configurations: PredefinedApiKeyOptions[];
 };
 //#endregion
-export { API_KEY_ERROR_CODES, API_KEY_TABLE_NAME, ApiKey, ApiKeyConfigurationOptions, ApiKeyOptions, apiKey, defaultKeyHasher };
+export { API_KEY_ERROR_CODES, API_KEY_TABLE_NAME, ApiKey, ApiKeyConfigurationOptions, ApiKeyOptions, PredefinedApiKeyOptions, apiKey, defaultKeyHasher };
 //# sourceMappingURL=index.d.mts.map
\ No newline at end of file
